free-3.md

```diff
@@ -10,41 +10,98 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+_Right now, without certain access and friends, it is difficult to play
+different types of card games competitively (Magic, Pokemon, Netrunner,
+etc.) because someone needs to buy the cards, which are not cheap, and
+practice playing, which requires someone else. Currently, most people
+choose not to play, for financial reasons or for lack of people to play
+with. Hopefully, this DSL would get them excited enough that they could 
+seek out people to play with because they would already know the game._
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+_These games have wide range of the types of cards that that can have. So,
+having a DSL allows the user to have variable rules and variable cards. It
+can do so by loading in the set of cards that the user will have, the set
+of cards that the computer will have, and play the game from there._
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+_It excites me that more people could become involved in these card games,
+so it can hopefully stop being only a nerdy elite who know how to play and
+can teach others to play. Additionally, the number of elements involved
+in creating this project would be interesting, from loading the different
+rules in, to the different cards in, to actually having a computer play
+with a person would be interesting. I came up with it by reading Prof Ben's
+idea generation page. When he wrote down card games, I thought about my
+experience trying to learn Magic. I was not interested in thinking about
+the theory of the game because I was so new. However, it is such a 
+strategy based game that without understanding the theory and strategy of
+Magic, it's difficult to do well. I ended up playing with my brother,
+who was happy destroying me without a question, but if I had tried to
+play with a more experienced crowd I did not know, I would have felt like
+an outsider because of my lack of experience._
 
 ### Domain
 _Describe the project's domain in five words._
 
+_Teaching card game novices basics_
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+_The user would have to load their cards into the program and tell the language
+which version of the rules it ants to use. Programming looks like changing the
+cards and rules to be able to play a game against the computer. This is the
+right way to interact with the problem domain because there are so many cards
+for any given game that it would be ridiculous to include all of them.
+Additionally, card developers release new cards very often to continue to make
+a profit, so a designer could not account for every single card. However, each
+card has a certain template that it follows. So, once the language learns
```

Comment:
However, games like YuGiOh add brand new features and tactics very often, such that I think it would be very hard to write a DSL that can handle all of that. See comment above.